models/renalStructures_CECT_segmentation/LICENSE

```diff
@@ -0,0 +1,201 @@
+ Apache License
```

Comment:
This line may need to be adjusted as shown in:
https://www.apache.org/licenses/LICENSE-2.0
```
                                 Apache License
                           Version 2.0, January 2004
                        http://www.apache.org/licenses/
```